etc/ci/performance/README.md

```diff
@@ -0,0 +1,58 @@
+Servo Page Load Time Test
+==============
+
+# Prerequisites
+
+* Python3
+
+# Usage
+## Build Servo
```

Comment:
Please put this section after `Prepare the test runner`, as that only needs to be done once but this has to be re-done for each Servo commit (nightly).